apps/frontend/src/app/PageArtifact/ArtifactCard.tsx

```diff
@@ -100,18 +104,18 @@ export default function ArtifactCard({ artifactId, artifactObj, onClick, onDelet
           </IconButton>}
           <Box sx={{ pt: 2, px: 2, position: "relative", zIndex: 1 }}>
             {/* header */}
-            <Box component="div" sx={{ display: "flex", alignItems: "center", gap: 1, mb: 1 }}>
+            <Box component="div" sx={{ display: "flex", alignItems: "center", gap: 0.4, mb: 1 }}>
               <Chip size="small" label={<strong>{` +${level}`}</strong>} color={Artifact.levelVariant(level)} />
               {!slotName && <Skeleton variant="text" width={100} />}
-              {slotName && <Typography noWrap sx={{ textAlign: "center", backgroundColor: "rgba(100,100,100,0.35)", borderRadius: "1em", px: 1 }}><strong>{slotName}</strong></Typography>}
-              {!slotDescTooltip && <Skeleton width={10} />}
-              {slotDescTooltip}
+              {slotName && <Typography noWrap sx={{ textAlign: "center", backgroundColor: "rgba(100,100,100,0.35)", borderRadius: "1em", px: 1.5 }}><strong>{slotName}</strong></Typography>}
+              {!slotDescTooltip ? <Skeleton width={10} /> : slotDescTooltip}
             </Box>
-            <Typography color="text.secondary" variant="body2">
-              <SlotNameWithIcon slotKey={slotKey} />
+            <Typography color="text.secondary" variant="body2" sx={{ display: "flex", gap: 0.5, alignItems: "center" }}>
+              <SlotIcon iconProps={{ fontSize: "inherit" }} slotKey={slotKey} />
+              {t<string>(`slotName.${slotKey}`)}
             </Typography>
             <Typography variant="h6" sx={{ display: "flex", alignItems: "center", gap: 1 }}>
-              {mainIcon}
+              <ColorText color={ele}><StatIcon statKey={mainStatKey} /></ColorText>

```

Comment:
Center aligning seems to not be working, looks like these are bottom aligned
![image](https://user-images.githubusercontent.com/36019388/215016682-7915ce91-e7d8-405c-b0af-3cdd44b202ab.png)